### DISCUSSION TITLE
[Question]: Using a Chromebook as a monitor

### DISCUSSION THREAD

**Discussion by ColoradoMark** [blakeblackshear/frigate#13420] [Ask A Question]
### What is your question?

I am running 0.14 and I like many of the new features. The log files show more errors than with the earlier version, but none seem fatal. One challenge I have is I created a group with 9 of my outdoor cameras. If I select that group on my PC they all show as expected all day. If I do the same on my Chromebook some of them blank out after a while. I have played with the Automatic Live View and Play Alert video settings. Having both on works best, but I still have the problem after a while only on the Chromebook. If I chose Birdseye on the Chromebook the cameras all show fine all day. What is the correct way to have the group display all day on the Chromebook when motion is not frequent?

**Comment by NickM-27** [comment]
There have been fixes for live view in the dev branch that might improve this

**Reply by ColoradoMark** [reply]
ok, I will install it when it is available to testers. Thanks.

**Reply by NickM-27** [reply]
There are several builds available now

**Reply by ColoradoMark** [reply]
Sorry, but I am not sure how to find them. I have been going to this site  https://github.com/blakeblackshear/frigate/releases where I installed 0.14.0 assuming it was the lastest for testing.

**Reply by NickM-27** [reply]
https://github.com/blakeblackshear/frigate/pkgs/container/frigate

**Reply by ColoradoMark** [reply]
I get permission denied no matter what I try. 

Sorry I am getting pretty good with Frigate and Home  Assistant with 14 cameras, but only just learning Github so I might be doing something wrong.

**Reply by hawkeye217** [reply]
Frigate 0.14.1 was just released: https://github.com/blakeblackshear/frigate/releases 

Let us know if your issues are fixed with this version.

**Comment by ColoradoMark** [comment]
So far it 0.14.1 seems to have corrected the problem.

Mark Endry
***@***.***

**Reply by hawkeye217** [reply]
Closing this. Feel free to reopen if you encounter the issue again.
